src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,31 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). Please add a private access modifier to `loadCapacity`.